substrafl_examples/get_started/plot_substrafl_torch_fedavg.py

```diff
@@ -3,16 +3,12 @@
 Using Torch FedAvg on MNIST dataset
 ===================================
 
-This example illustrates the basic usage of SubstraFL, and proposes Federated Learning model training using the Federated Average strategy
+This example illustrates the basic usage of SubstraFL and proposes Federated Learning using the Federated Averaging strategy
 on the `MNIST Dataset of handwritten digits <http://yann.lecun.com/exdb/mnist/>`__ using PyTorch.
-In this example, we work on 28x28 pixel sized grayscale images. The problem considered is a classification problem
-aiming to recognize the number written on each image..
+In this example, we work on 28x28 pixel sized grayscale images. This is a classification problem
+aiming to recognize the number written on each image.
 
-SubstraFL can be used with any machine learning framework (PyTorch, Tensorflow, Scikit-Learn, etc). However a specific
-interface has been developed for PyTorch which makes writing PyTorch code simpler than with other frameworks.
-This example here uses the specific PyTorch interface.
-
-This example does not use a deployed platform of Substra and run in local mode.
+This example does not use a deployed platform of Substra and runs in local mode.
 
```

Comment:
I felt that this was already mentioned several times throughout the docs and is not necessary to move through the example